### ISSUE TITLE
does order really matter?

### ISSUE THREAD

**Issue by fraro01** [Alfrick/Create-Cryptocurrency-in-Python#4]
Hello Alfrick!
A question came up to me while I was reading your code and the corrections you wrote:

"the order of arguments switched", does it really matters? 
Because it seems working even without correcting the order.

**Comment by Alfrick** [comment]
Yes, it works both ways. But it's good to stick to the conventional practice.

**Comment by fraro01** [comment]
Thanks a lot!
Another question that I have, regards the method you called: "create_node".
what does it do? 
I was thinking if we could use it with the set called "nodes" (initialized in the: "__init__"), just to add the participants of the blockchain to the set: "nodes".
For instance, in the method: "new_data" in the end I've writen two lines of code, so that every time I add a transaction, if the sender and/or the receiver aren't in the "node-set" I'm gonna add them. 

And one more question:
the method: "obtain_block_object(block_data)" how can it be useful?

thanks a lot and sorry if bother you :)
